src/views/components/LoginForm/container.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { withFormik } from 'formik'
+import * as yup from 'yup'
+
+import handleSubmit from 'Modules/formikEvents/handleSubmit'
+import LoginFormComponent from './component'
+import { defaultProps, propTypes } from './types'
+
+const LoginForm = ({ isSubmitting, status }) => (
+  <LoginFormComponent
+    status={status}
+    isSubmitting={isSubmitting}
+  />
+)
+
+LoginForm.propTypes = {
+  ...propTypes
+}
+
+LoginForm.defaultProps = {
+  ...defaultProps
```

Comment:
useless spread

Reply:
Not useless, it shared types, I add it to component

Reply:
can we do `LoginForm.defaultProps = defaultProps` ?

Reply:
In spread case editor help to highlight types) And if we have different specific props, we can easy add additional type